lib/plugins/backup.rb

```diff
@@ -92,15 +123,21 @@ def zip(io_tar)
         zip_stream
       end
 
+      # Uploads an object from the pass
```

Comment:
*passed -- what column are we wrapping these comments to? I think 80 is a good standard.